data/ui/notes.ui

```diff
@@ -54,7 +56,7 @@
                     </style>
                     <child>
                       <object class="GtkSearchEntry" id="search_entry">
-                        <property name="placeholder-text" translatable="true">Search...</property>
```

Comment:
`…` should be used instead of tree dots (according to HIG)

Reply:
It was already like this that before the change, but I'll change the three dots to an ellipsis char.